packages/sit-onyx/src/components/OnyxBadge/OnyxBadge.vue

```diff
@@ -0,0 +1,94 @@
+<script lang="ts" setup>
+import { useDensity } from "../../composables/density";
+import type { OnyxBadgeProps } from "./types";
+
+const props = withDefaults(defineProps<OnyxBadgeProps>(), {
+  variation: "primary",
+});
+
+const { densityClass } = useDensity(props);
+
+defineSlots<{
+  /**
+   * Badge content.
+   */
+  default(): unknown;
+}>();
+</script>
+
+<template>
+  <div class="onyx-badge" :class="[`onyx-badge--${props.variation}`, densityClass]">
+    <div class="onyx-badge--content onyx-truncation-ellipsis">
+      <slot></slot>
+    </div>
+  </div>
+</template>
+
+<style lang="scss">
+@use "../../styles/density.scss";
+
+.onyx-badge {
+  @include density.compact {
+    --onyx-badge-padding: 0 var(--onyx-spacing-2xs);
+    --onyx-badge-gap: var(--onyx-spacing-4xs);
+    --onyx-badge-height: 1.5rem;
+  }
+
+  @include density.default {
+    --onyx-badge-padding: var(--onyx-spacing-5xs) var(--onyx-spacing-sm);
+    --onyx-badge-gap: var(--onyx-spacing-2xs);
+    --onyx-badge-height: 1.75rem;
+  }
+
+  @include density.cozy {
+    --onyx-badge-padding: var(--onyx-spacing-4xs) var(--onyx-spacing-sm);
+    --onyx-badge-gap: var(--onyx-spacing-sm);
+    --onyx-badge-height: 2rem;
+  }
+
+  --onyx-badge-background-color: var(--onyx-color-base-primary-500);
+
+  display: inline-flex;
+  height: var(--onyx-badge-height);
+  max-width: 100%;
+  box-sizing: border-box;
+  padding: var(--onyx-badge-padding);
+  justify-content: center;
+  align-items: center;
+  gap: var(--onyx-badge-gap);
```

Comment:
those don't seem to make any difference.
<img width="231" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/15b51830-3631-42ba-9e46-368a5d00e6f8">
```suggestion
```